docs/external_system.md

```diff
@@ -250,114 +272,67 @@ steps needed to configure SAT to use externally-accessible API endpoints exposed
 
 ## Authenticate SAT Commands
```

Comment:
I'm looking at this more closely now, and I'm realizing that most of the after this point needs to be removed or  modified in order to make sense in the context of configuring SAT on an external system. In the previous steps in the section "Install and Configure SAT", we have already:

* Run `sat init` to generate a SAT config file (Step 11)
* Modified the configuration file to set the API gateway hostname and the s3 endpoint (Step 13)
* Modified the username in the SAT config file (Step 14)

All that really remains to do here is:

* Run `sat auth`. This is just step 4 of the "Authenticate SAT Commands" section
* Run steps 1 and 2 of "Generate SAT S3 Credentials" section below. This works because we set up our Kubernetes config file in steps 7-10 of the "Install and Configure SAT" procedure.

What do you think about adding these remaining things as additional steps in this "Install and Configure SAT" procedure? The reason I suggest doing it this way instead of preserving the two following sections is that we have already run `sat init`, and that's really integral to some of the other steps in the section, and we've also already set the username in the SAT config file.

Reply:
Sounds like a great idea! I'll update in the next commit.